steam/core/cm.py

```diff
@@ -509,7 +521,36 @@ def str_to_tuple(serveraddr):
         self.merge_list(map(str_to_tuple, serverlist))
 
         return True
+    
+    def bootstrap_from_webapi_websocket(self):
+        """
+        Fetches CM server list from WebAPI and replaces the current one
+
+        :return: booststrap success
+        :rtype: :class:`bool`
+        """
+        self._LOG.debug("Attempting bootstrap via WebAPI for websocket")
 
+        from steam import webapi
```

Comment:
Why are you doing an import in a function?

Reply:
It's from the original `bootstrap_from_webapi` function, do you want me to change it in this pr?
https://github.com/ValvePython/steam/commit/5f11d6d7cc15ba0eb810150bd5cd074fbab8840f#diff-1c8fa68928ae302603b5f8fa5bf7ca188dfcf186946269b33f367317bb5b2e68R443

Reply:
That's kinda weird...we'll leave it if it's something the original dev did. I'm not familiar enough with this codebase to categorically say it's without good reason.